README.md

```diff
@@ -146,6 +148,23 @@ Another example is the tutorial for the SARS-CoV-2 workflow, which lives alongsi
 Subprojects necessitate a distinct URL, such as https://docs.nextstrain.org/projects/augur as opposed to just https://docs.nextstrain.org.
 By default, this means a distinct set of headings / links in the sidebar navigation, making navigating back to https://docs.nextstrain.org more difficult once you have navigated to a subproject.
 
+## Building the SubProject docs
+
+Typically the documentation for each subproject is build using the same commands as the umbrella docs described above:
+```
+cd <repo>/docs
+make clean # optional
+make html
+```
```

Comment:
I would omit these pseudo-instructions since they're not specific to a project and just keep the (below) links out to each subproject's own how-to for building its docs.

Reply:
Resolved in 77f7f74

Reply:
It would be nice to have consistency (e.g. all directories named `docs`, all using `make html`) but I don't have the appetite to go changing that in a bunch of repos right now!